components/login/passwordReset/src/hooks/components/useGetStageInfoStep.js

```diff
@@ -0,0 +1,52 @@
+import {STATUSES} from '@s-ui/react-molecule-progress-steps'
+
+import {
+  STAGE_PASSWORD_CHANGE,
+  STAGE_PASSWORD_RESET_START
+} from '../../config.js'
+import useGetCurrentToken from '../useGetCurrentToken.js'
+import useI18n from '../useI18n.js'
+const useGetStageInfoStep = () => {
+  const i18n = useI18n()
+  const {getCurrentToken} = useGetCurrentToken()
+  const {stage} = getCurrentToken()
+
+  const getStageInfoData = () => {
+    const title =
+      stage === STAGE_PASSWORD_RESET_START
+        ? i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_1.TITLE')
+        : i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_2.TITLE')
+
+    const message =
+      stage === STAGE_PASSWORD_RESET_START
+        ? i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_1.MESSAGE')
+        : i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_2.MESSAGE')
+
+    const initialStepperState = {
+      label: i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_1.LABEL'),
+      status:
+        stage === STAGE_PASSWORD_RESET_START
+          ? STATUSES.ACTIVE
+          : STATUSES.VISITED
+    }
+
+    const endStepperState = {
+      label: i18n.t('LOGIN_CROSS.PASSWORD_RESET.STEP_2.LABEL'),
+      status:
+        stage === STAGE_PASSWORD_CHANGE ? STATUSES.ACTIVE : STATUSES.NORMAL
+    }
```

Comment:
IMHO, it could be cleaner to have one single condition to determine which is the current stage, and then create an object with title, message, initialStepperState and endStepperState properties for each kind of state.

Something like this...

```
if (stage === 1) {
   data = {
     title: '1',
     message: '1'
   }
} else {
   data = { title: '2', message: '2' }
}

return data
```